routes/loginRouter.js

```diff
@@ -1,6 +1,8 @@
 const userController = require('../controllers/userController');
 var router = require('express').Router();
 
-router.get("/login", userController.getLoginPage);
+router.get("/user/login", userController.getLoginPage);
 
-module.exports = router;
\ No newline at end of file
+// router.post("/login",userController.getThanksPage);
```

Comment:
what is this outcommented line for exactly? should you wait for BE or smth?

Reply:
no the login actually needs no thank you, it will just direct to the profile page of the user.
The sign up can have this thank you page after they sign up btw.

Reply:
yes yes, I was just confused, thank you for response